src/nodejs/integration/nodejs_app_with_seeker_test.go

```diff
@@ -52,6 +52,8 @@ var _ = Describe("CF NodeJS Buildpack", func() {
 			Expect(app.Restart()).To(Succeed())
 			Eventually(func() ([]string, error) { return app.InstanceStates() }, 20*time.Second).Should(Equal([]string{"RUNNING"}))
 
+			// test that Seeker was required by buildpack
+			Expect(app.Files("app/dist/server.js")).To(ContainElement("require('/home/vcap/app/seeker/node_modules/@synopsys-sig/seeker');"))
```

Comment:
It seems like `app.Files` lists the files in app workspace, not the file contents. It runs a `find` not a `grep`.

Reply:
fixed